components/cli/cli.gradle

```diff
@@ -14,18 +14,12 @@ repositories {
 }
 
 dependencies {
-    compile(group: 'ome', name: 'formats-gpl', version: '5.5.0'){
-    }
-    compile(group: 'ome', name: 'formats-bsd', version: '5.5.0'){
-    }
-    compile(group: 'org.springframework.ldap', name: 'spring-ldap', version: '1.3.0.RELEASE', classifier: 'all'){
-        exclude group: 'org.testng', module: 'testng'
-    }
-    compile(group: 'omero', name: 'blitz', version: '5.3.2-ice36-b62') {
+    compile(group: 'org.openmicroscopy', name: 'omero-gateway', version: '5.5.4') {
```

Comment:
```suggestion
    compile(group: 'org.openmicroscopy', name: 'omero-gateway', version: '5.6.2') {
```

Reply:
I made the change locally and run with 5.6.2 but I forgot to push the commit :-)